subsys/bluetooth/controller/ll_sw/nordic/lll/lll_peripheral_iso.c

```diff
@@ -403,6 +403,21 @@ static void abort_cb(struct lll_prepare_param *prepare_param, void *param)
 
 		cis_lll = ull_conn_iso_lll_stream_get_by_group(cig_lll, NULL);
 
+		/* FIXME: Consider Flush Timeout when resetting current burst number */
+		if (!has_tx) {
+			has_tx = 1U;
+
+			/* Adjust nesn when flushing Tx */
+			/* FIXME: When Flush Timeout is implemented */
+			if (bn_tx <= cis_lll->tx.bn) {
+				/* sn and nesn are 1-bit, only Least Significant bit is needed */
+				cis_lll->sn += cis_lll->tx.bn + 1U - bn_tx;
```

Comment:
Can you elaborate on the `+ 1U` here?

Reply:
bn_tx start at 1. Hence, it is pretty much this:

```suggestion
				cis_lll->sn += cis_lll->tx.bn - (bn_tx - 1U);
```

Reply:
There quite a bit of duplicate code in LLL implementation, as the development has been iterative. Hope to refactor all at once, when starting to cover conformance tests.